chaos-monkey-spring-boot/src/main/java/de/codecentric/spring/boot/chaos/monkey/component/ChaosMonkeyScheduler.java

```diff
@@ -76,14 +76,14 @@ private void cancelOldTasks(Map<ChaosMonkeyRuntimeAssault, String> cronExpressio
     }
 
     private void scheduleNewTasks(Map<ChaosMonkeyRuntimeAssault, String> cronExpressions) {
+        Logger.info("Schedule  %s cron task(s)".formatted(cronExpressions.size()));
```

Comment:
Is this new statement here intentionally or leftover from debugging?

Reply:
Somewhat intentionally. The logger was unused and before deleting it, we might can get some good usage of it.